src/main/java/org/example/Operations/CollectionOperations.java

```diff
@@ -33,7 +33,7 @@ public Response create(String token, String lpath, OptionalInt createIntermediat
          formData.put("op", "create");
          formData.put("lpath", lpath);
          if (createIntermediates.isPresent()) {
-             formData.put("create-intermediates", String.valueOf(createIntermediates));
+             formData.put("create-intermediates", String.valueOf(createIntermediates.getAsInt()));
```

Comment:
I noticed some lines use `.ifPresent` while others use an if-stmt.

Let's pick one style and stick with that unless there's a good reason to use a different style. _This is only a recommendation._

Reply:
When I went changed the way I implemented optional params I completely forgot that I was using `.ifPresent`. I've changed everything to use `.ifPresent` now.